publiccode.yml

```
@@ -0,0 +1,93 @@
+publiccodeYmlVersion: '0.2'
+name: SBNWeb-Open
+releaseDate: '2019-03-11'
+softwareVersion: v1.0.0-open
+url: 'https://github.com/IstitutoCentraleCatalogoUnicoBiblio/SBNWeb-Open'
+developmentStatus: stable
+softwareType: standalone/web
+platforms:
+  - web
+categories:
+  - library-management-system
+  - collaboration
+  - communications
+maintenance:
+  type: contract
+  contractors:
+    - name: 'Almaviva'
+      until: '2021-05-31'
+legal:
+  license: AGPL-3.0-or-later
+  mainCopyrightOwner: 'Istituto Centrale per il Catalogo Unico delle biblioteche italiane e per le informazioni bibliografiche'
+  repoOwner: 'Istituto Centrale per il Catalogo Unico delle biblioteche italiane e per le informazioni bibliografiche'
+intendedAudience:
+  scope:
+    - library management
```

Anche per quanto riguarda il campo `intendedAudience/scope`, c'è una lista controllata dalla quale attingere: https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/scope-list.html#scope-list
Per questo motivo suggerirei:
```suggestion
    - culture
    - research
    - science-and-technology
```
oppure altri campi tratti da quella lista. Grazie molte.

---

ok, l'ho modificato con i due tag che vedi